File: C++/C3 10 AS 04 More on Loops/04.cpp
//04. Write a C++ program to calculate LCM of two numbers.
#include<iostream>
using namespace std;
int main()
{
    int a, b, i;
    cout<<"Enter two numbers: ";
    cin>>a>>b;
    for(i=1;i<=a*b;i++)
        if(i%a==0&&i%b==0)
            break;
    cout<<"LCM of "<<a<<" & "<<b<<" is: "<<i;
    return 0;
}

File: Python/PY-18-AS-07-Decision-Control/04.py
# 04. Write a python script to print greater between two numbers. Print number only once if both the numbers are same.
var1 = int(input("Enter 1st number: "));
var2 = int(input("Enter 2nd number: "));
if var1 > var2:
    print("%d is Greater"%(var1));
elif var2 == var1:
    print("%d Both are same"%(var1));
else:    
    print("%d is Greater"%(var2));
